common/rdr/HexOutStream.cxx

```diff
@@ -41,7 +40,7 @@ bool HexOutStream::flushBuffer()
 {
   while (sentUpTo != ptr) {
     uint8_t* optr = out_stream.getptr(2);
-    size_t length = min(ptr-sentUpTo, out_stream.avail()/2);
+    size_t length = std::min(ptr-sentUpTo, (int64_t)out_stream.avail()/2);
```

Comment:
`int64_t` seems like an odd choice here as neither the left or right side as that type. Why not `size_t` or `ptrdiff_t`?